app/models/project.rb

```diff
@@ -104,28 +115,16 @@ def departments
   end
 
   def project_directory
```

Comment:
`project_directory`, `project_directory_parent_path`, and `project_short` now behave different because (1) we force all projects to be under `/tigerdata` (that's not the parent path for all projects) and (2) we allow users to enter a deep path (e.g. "my-department/my-project) as part of the project or a single path (e.g. "my-project")